libs/blocks/tabs/tabs.js

```diff
@@ -84,89 +87,109 @@ function initTabs(elm, config, rootElem) {
   if (config) configTabs(config, rootElem);
 }
 
-let initCount = 0;
+function handleDeferredImages(e) {
+  const images = e.querySelectorAll('img[loading="lazy"]');
+  images.forEach((img) => {
+    /* c8 ignore next */
+    img.removeAttribute('loading');
+  });
+  e.removeEventListener('milo:deferred', handleDeferredImages, true);
+}
 
 const init = (block) => {
   const rootElem = block.closest('.fragment') || document;
   const rows = block.querySelectorAll(':scope > div');
+  const parentSection = block.closest('.section');
   /* c8 ignore next */
   if (!rows.length) return;
+  block.addEventListener('milo:deferred', handleDeferredImages(block), true);
```

Comment:
This runs `handleDeferredImages` right away.  Also the `milo:deferred` event is only fired on the `document`. and not each individual block.  Something like:

```suggestion
  handleDeferredImages(block);
```

```
const handleDeferredImages = (block) => {
  const loadLazyImages = () => {
    const images = block.querySelectorAll('img[loading="lazy"]');
		  images.forEach((img) => {
		    /* c8 ignore next */
		    img.removeAttribute('loading');
		  });
		  document.removeEventListener('milo:deferred', loadLazyImages, true);
  };
  document.addEventListener('milo:deferred', loadLazyImages, true);
}
```

Reply:
Might be premature optimization, but any images on a tab that is not currently displayed can have the image loading left to lazy.

Reply:
@chrischrischris  - Thanks for the feedback on this. I've addressed this file for now. 
LMK if we want to add a `milodefer` func to utils.

Reply:
No need to add `milodefer` for now until we have more use cases.